### PR TITLE
Clarify documentation regarding `+bundled`

### PR DESCRIPTION
**Pull request by In-line**

### CODE REVIEW

**Review by weihanglo — COMMENTED** [review]
Thanks for the pull request!

Should we put this in [Rustc Book](https://doc.rust-lang.org/stable/rustc/command-line-arguments.html#-l-link-the-generated-crate-to-a-native-library) instead? From my understanding Cargo passes it as arg value of `-l` flag to rustc.

https://github.com/rust-lang/cargo/blob/e07efd8eaa0ca74aa6cac31d760c8d50e6ed2b07/src/cargo/core/compiler/mod.rs#L492-L494

**Comment by In-line** [comment]
I think ideally it should be duplicated in both places, just because it's very confusing point. We had to debug for few weeks a linking issue at the company I'm currently working, just because someone forgot `static` in one place 😅 .

File: src/doc/src/reference/build-scripts.md [hunk 1]
```diff
@@ -183,7 +183,11 @@ through the library target's public API.
 The optional `KIND` may be one of `dylib`, `static`, or `framework`. See the
 [rustc book][option-link] for more detail.
 
+Note that if you don't specify the `KIND` when using `rustc-link-lib`, the default linkage modifiers won't be applied. 
+For instance, [`+bundled` linkage modifier][bundled-link] won't be used in conjunction with `static` by default.
```
Comment (sanmai-NL) [inline src/doc/src/reference/build-scripts.md:187]: .., the [+bundled` ...

**Comment by weihanglo** [comment]
Closing this as inactive. Feel free to reöpen this pr or create a new pr if you get the time to work on this. Thanks

**Pull request closed without merging**